dist/client/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html><html lang="en" translate="no"><head><title>Silex</title><meta name="description" content="This is a public instance of Silex website builder. Build websites in the browser"><link rel="shortcut icon" href="assets/favicon.png?357271"><style>body {
+<!DOCTYPE html><html lang="en" translate="no"><head><title>Silex</title><meta name="description" content="This is a public instance of Silex website builder. Build websites in the browser"><link rel="shortcut icon" href="assets/favicon.png?596140"><style>body {
```

Comment:
pls don't commit this, just revert the change and commit the reverted change
(use git revert or just put back the initial value)